### PR TITLE
Apps

### PR DESCRIPTION
**Pull request by dannyclever**
This is the first Pull Request

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,929 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <!-- <link rel="stylesheet" href="reset.css" /> -->
+    <link rel="stylesheet" href="test.css" />
```
Comment (samgaco) [inline index.html]: That's not the correct name of the .css file you are using for this pull request.

**Review by samgaco — CHANGES_REQUESTED** [review]
### Hi @dannyclever and @macnick :hand: 

### Everything is looking great and the readme file looks nice :clap: , but there are some blemishes and things to adjust:

* There is a **horizontal scrolling bar** in the small screen version that shouldn't go there.

* In the **apps section until the insights section** you have the following problem for the small screen (the images are stacking to close for the first sections):
tnw1

* The section **Latest News** is not adapted well for the small screen:
tnw2


* For the large screen size, the **vertical borders** (left and right) should be a bit wider, like in the original.
Alt Text

* You have to **solve the conflict** with the README.md file before the pull request.

### Work on these things and submit back for another code review :pray:

**Comment by bolah2009** [comment]
Currently been reviewed

File: style.css [hunk 2]
```diff
@@ -0,0 +1,1004 @@
...
+.section-latest-title a:hover {
+  text-decoration: underline;
+}
+h3 {
+  position: relative;
+  top: -20px;
+  font-size: 20px;
+}
+.section-popular-byline {
+  /* display: block; */
```
Comment (bolah2009) [inline style.css:538]: `SUGGESTION`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,1004 @@
...
+  .deals-header {
+    flex-direction: column;
+  }
+  .story-deal-discount {
+    display: none;
+  }
+  /* .story-deal {
+    display: block;
+    flex-direction: row;
+  } */
```
Comment (bolah2009) [inline style.css:933]: `SUGGESTION`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

**Review by bolah2009 — APPROVED** [review]
## Nice Job Team :clap:

The [clone](https://raw.githack.com/dannyclever/Responsive-Design-/apps/index.html) looks good to me and is responsive. :+1: 


#### Check out the comment below and [these errors](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2Fdannyclever%2FResponsive-Design-%2Fapps%2Findex.html) on using an heading (h2-h6) element in an `<article>` and `<section>` elelment, using `alt` for `` elements and others. Consider checking them out before merging.

Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**